src/ui_buttons.cpp

```diff
@@ -4,9 +4,9 @@
 #include "arp.h"
 #include "util.hpp"
 #include "ui_vars.h"
+#include <EEPROM.h>
```

Comment:
not here. The `ui_buttons.cpp` file as only one concern: handle the user interface and update the preset and other variables accordingly. Persisting these changes in EEPROM is not one of its concerns.